client/playwright.config.ts

```diff
@@ -17,6 +18,7 @@ const config = {
   ],
   use: {
     baseURL: 'http://localhost:4000/dtaas/',
+    trace: 'retain-on-failure',
```

Comment:
It works perfectly fine, as long as the `baseURL` is ending with `/`.

Also found a way to make this configuration read from `.env` instead of being hardcoded. But this would require a sizable change to the codebase.